bbb-common-web/src/main/java/org/bigbluebutton/api/domain/Meeting.java

```diff
@@ -363,6 +365,22 @@ public String getDefaultAvatarURL() {
 		return defaultAvatarURL;
 	}
 
+	public void setWaitingPositionsInWaitingQueue(String userId, String idx) {
+		if (guestUsersWithPositionInWaitingLine.containsKey(userId)) {
+			guestUsersWithPositionInWaitingLine.replace(userId,idx);
+		} else {
+			guestUsersWithPositionInWaitingLine.put(userId,idx);
+		}
+	}
+
+	public String getWaitingPositionsInWaitingQueue(String userId) {
+		if (guestUsersWithPositionInWaitingLine.containsKey(userId)) {
+			return guestUsersWithPositionInWaitingLine.get(userId);
+		} else {
+			return null;
+		}

```

Comment:
A miss in a `HashMap` `get` function will return `null` by default. Maybe you could use the `else` return something else or just return the `get` without testing if the key exists.